pkg/config/debugging.go

```diff
@@ -9,15 +16,32 @@ type Debugging struct {
 	LogLevel string `json:"logLevel"`
 }
 
-// GetVerbosity returns the numerical value for LogLevel which is an enum
-func (c *Config) GetVerbosity() int {
-	var levelNames = map[string]int{
-		"normal":   2,
-		"debug":    4,
-		"trace":    6,
-		"traceall": 8,
+var logLevelNames = map[string]int{
+	"normal":   2,
+	"debug":    4,
+	"trace":    6,
+	"traceall": 8,
+}
+
+// cleanLoggingSetting validates the logging setting and saves a
+// warning if there is an issue.
+func (c *Config) cleanLoggingSetting() {
```

Comment:
What do you think about changing this to `validateLoggingSetting()` or something similar? Cleaning gives the impression of something being removed

Reply:
That's a good point. I renamed it `computeLoggingSetting()` to match the other compute function and I moved the invocation into `updateComputedValues()` to align with when recomputed things end up changing the content of the data structure.